clickable/commands/build-libs.py

```diff
@@ -0,0 +1,62 @@
+import os
+import sys
+import subprocess
+import inspect
+import glob
+from os.path import dirname, basename, isfile, join
+
+from .base import Command
+from clickable.utils import print_warning
+from clickable.build_templates.base import Builder
+from clickable.container import Container
+from clickable.device import Device
+
+class LibBuildCommand(Command):
+    aliases = []
+    name = 'build-libs'
+    help = 'Compile the library dependencies'
+
+    def run(self, path_arg=None):
+        if not self.config.lib_configs:
+            print_warning('No libraries defined.')
+
+        self.container.setup_dependencies()
+
+        for lib in self.config.lib_configs:
+            dir_tmp = lib.dir
+            for arch in lib.architectures:
+                if arch in lib.arch_triplets:
+                    if not lib.custom_docker_image:
+                        lib.docker_image = 'clickable/ubuntu-sdk:16.04-{}'.format(arch)
```

Comment:
Until vivid support is dropped from clickable this should probably check if `is_xenial` and adjust accordingly

Reply:
Done